### DISCUSSION TITLE
Transfert Chia 3 XCH Send 1 XCH receiv...

### DISCUSSION THREAD

**Discussion by thangorodrim99** [Chia-Network/chia-blockchain#17306] [Support]
Translate by Deepl (sorry for my English) :

Hello,

At the beginning of January, I transferred 3 Chia from one portfolio (xch1gj27nlqhng4due5cn9rs6gq0uem5cpv4phy6dd9dv0pgt5f2gt6swrun8c) to another portfolio (xch1v55nerhdzew94dmqwwmgapat0z3w2ysdnjwc07m2npwunep5l0rsc7qzu6). The transfer took 3 days but was completed.

However, of the 3 XCH sent with fees, I only received 1 XCH without fees...

You will find attached the screenshots of the sending and receiving as well as the identical height value on both sides.

I remain at your disposal to provide you with more information on request, but I would like a solution to find the 2 XCH that have disappeared.

Original text :
Bonjour,

Début janvier, j’ai procédé à un transfert de 3 Chia d’un portefeuille (xch1gj27nlqhng4due5cn9rs6gq0uem5cpv4phy6dd9dv0pgt5f2gt6swrun8c) vers un autre portefeuille (xch1v55nerhdzew94dmqwwmgapat0z3w2ysdnjwc07m2npwunep5l0rsc7qzu6). Le transfert a mis 3 jours mais a été réalisé.

Cependant sur les 3 XCH envoyé avec des frais, je n’en ai reçu que 1 XCH sans frais…

Vous trouverez ci-joint les captures d’écran de l’envoi et de la réception ainsi que la valeur de hauteur identique de part et d’autre.

Je reste à votre disposition pour vous fournir plus d’information à la demande mais j’aimerai une solution pour retrouver les 2 XCH qui ont disparu.

![2024-01-12 12_01_33-Window](https://github.com/Chia-Network/chia-blockchain/assets/147111558/ab7fe10a-e142-4392-96f7-e2538c1c84f9)
![2024-01-12 12_02_08-Chia Blockchain](https://github.com/Chia-Network/chia-blockchain/assets/147111558/39b03949-09df-4a99-803a-9d98f511198e)

**Comment by BrandtH22** [comment]
Hey @thangorodrim99 , can you try resyncing the sending wallet? Upon reviewing the transactions what I am thinking happened is there was first an attempt to send 1 xch that was pending then the delete unconfirmed transactions was used and a second transaction of 3 xch was submitted. 

Since the delete unconfirmed transactions is a local only occurrence, what it seems happened is the initial transaction attempt for the 1 xch went through before the 3 xch attempt could. This made the 3xch transaction invalid and therefore not complete. 

The sending (originating) wallet may have become confused by this and reported that all 3 xch were sent when in reality only 1 was.

To verify if this is what happened what I would do is use the gui settings / advanced / resync option. This will close the gui and upon reopening will force the wallet to resync from scratch.

Let us know if the full balance is visible in the sending wallet after performing the resync, if not we can dive further into troubleshooting.

---
This content has been translated using translation software:
Ce contenu a été traduit à l'aide d'un logiciel de traduction :
---

pouvez-vous essayer de resynchroniser le portefeuille d'envoi? Après avoir examiné les transactions, je pense qu'il s'est d'abord passé une tentative d'envoi de 1 xch qui était en attente, puis la suppression des transactions non confirmées a été utilisée et une deuxième transaction de 3 xch a été soumise. 

Étant donné que la suppression des transactions non confirmées est une occurrence locale uniquement, il semble que la tentative de transaction initiale pour le 1 xch ait été effectuée avant la tentative de 3 xch. 

Cela a rendu la transaction 3xch invalide et donc incomplète. Le portefeuille émetteur (d'origine) a peut-être été dérouté par cela et a signalé que les 3 xch ont été envoyés alors qu'en réalité un seul l'a été. 

Pour vérifier si c'est ce qui s'est passé, ce que je ferais est d'utiliser l'option paramètres de l'interface graphique / avancé / resynchronisation. Cela fermera l'interface graphique et, lors de la réouverture, forcera le portefeuille à se resynchroniser à partir de zéro. 

Faites-nous savoir si le solde complet est visible dans le portefeuille d'envoi après avoir effectué la resynchronisation, sinon nous pouvons aller plus loin dans le dépannage.

**Comment by thangorodrim99** [comment]
Translate by Deepl (sorry for my English) :

It's good to know that requested transactions remain in progress even if you've asked for unconfirmed transactions (which are therefore only local) to be deleted. I'll have to check closely what happens next, as I've actually done this several times when the system has told me it can't charge during the transaction.

The sender and receiver portfolios have the right total values, even if there's still a visual error on the sender portfolio (the 3 XCH transaction persists but doesn't seem to be considered in the final sum). 28 XCH cultivated with 11 transferred, I still have 17 on this account.

Original text :

C'est bon à savoir que les transactions demandées restent en cours même si on a demandé la suppression des transactions non confirmées (qui ne sont donc qu'en local). Il va falloir que je vérifie de près la suite car j'ai effectivement fait cela plusieurs fois lorsque le système m'a indiqué ne pas pouvoir faute de frais durant la transaction.

Le portefeuille émetteur et receveur ont bien les bonnes valeurs totales même s'il reste une erreur visuelle sur le portefeuille émetteur (la transaction de 3 XCH persiste mais ne semble pas considérée dans la somme finale). 28 XCH cultivés avec 11 transférés il m'en reste bien 17 sur ce compte.

![2024-01-13 17_50_50-Chia Blockchain](https://github.com/Chia-Network/chia-blockchain/assets/147111558/9b384f50-4784-4bf2-a303-cb1e30fffca0)

**Reply by BrandtH22** [reply]
Let us know if any other questions arise 👍 
You should be able to perform a resync via the GUI to have the balance and transactions updated properly (settings / advanced / resync). This will close the gui and upon reopening will fully resync the wallet which should update balances and transactions to reflect the current status.

This content has been translated using translation software:
Ce contenu a été traduit à l'aide d'un logiciel de traduction :

Faites-nous savoir si d'autres questions se posent 👍 Vous devriez être en mesure d'effectuer une resynchronisation via l'interface graphique pour que le solde et les transactions soient correctement mis à jour (paramètres / avancé / resynchronisation). Cela fermera l'interface graphique et, lors de la réouverture, resynchronisera complètement le portefeuille, qui devrait mettre à jour les soldes et les transactions pour refléter l'état actuel.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
